File: Super easy/UVa12279.py
c = 1
while True:
	n = int(input())
	if n == 0:
		break
	zeros = 0
	non_zeros = 0
	x = map(int,input().split())
	for i in x:
		if i != 0:
			non_zeros += 1
		else:
			zeros += 1
	print("Case %d: %d"%(c,non_zeros-zeros))
	c += 1

File: Super easy/Uva12279.cpp
#include<iostream>

using namespace std;

int main()
{
	int c = 1;
	while(1)
	{
		int n;
		cin >> n;
		if(!n)break;
		int zeros=0,non_zeros=0;
		for(int i=0;i<n;i++)
		{
			int input;
			cin >> input;
			if(input)non_zeros++;
			else zeros++;
		}
		cout << "Case " << (c++) << ": " << (non_zeros-zeros) << endl;
	}
	return 0;
}
